Tiqr/tiqr-app-core-android
# Bugfix/tiqr 327 328

### Pull Request Checklist

### Short Description of Change
Post testing the fix for TIQR-327 made the screen for the login confirmation require extra padding as well between the `Account Details` and the information.

Fix for crash
⚠️ 🛑  **When the app integrate this core the custom application code must include the line `TiqrConfig.initialize(this)` in its initialization** ⚠️ 🛑
The TIQR & eduID apps must be responsible for the initialization of the TiqrConfig.
The TiqrConfig must be initialized as soon as the application process is created.
The MainActivity cannot perform this initialization because it has a ViewModel injected, which in turn relies on the initialization of the TiqrConfig. Since the viewModel is injected as a field, the execution and completion of the MainActivity's onCreate method is not guaranteed before the field initialization.

```
Exception java.lang.RuntimeException:
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:4166)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:4312)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:101)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2571)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loopOnce (Looper.java:226)
  at android.os.Looper.loop (Looper.java:313)
  at android.app.ActivityThread.main (ActivityThread.java:8741)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:571)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1067)
Caused by kotlin.UninitializedPropertyAccessException:
  at org.tiqr.data.model.TiqrConfig.getAuthScheme (TiqrConfig.java)
  at org.tiqr.data.repository.AuthenticationRepository.<init> (AuthenticationRepository.java)
  at org.tiqr.data.module.RepositoryModule.provideAuthenticationRepository$data_release (RepositoryModule.java)
  at org.tiqr.data.module.RepositoryModule_ProvideAuthenticationRepository$data_releaseFactory.provideAuthenticationRepository$data_release (RepositoryModule_ProvideAuthenticationRepository.java)
  at org.tiqr.DaggerTiqrApplication_HiltComponents_SingletonC$SingletonCImpl$SwitchingProvider.get (DaggerTiqrApplication_HiltComponents_SingletonC.java)
  at dagger.internal.DoubleCheck.get (DoubleCheck.java)
  at org.tiqr.DaggerTiqrApplication_HiltComponents_SingletonC$ViewModelCImpl$SwitchingProvider.get (DaggerTiqrApplication_HiltComponents_SingletonC.java)
  at dagger.hilt.android.internal.lifecycle.HiltViewModelFactory$1.create (HiltViewModelFactory.java)
  at androidx.lifecycle.AbstractSavedStateViewModelFactory.create (AbstractSavedStateViewModelFactory.java)
  at dagger.hilt.android.internal.lifecycle.HiltViewModelFactory.create (HiltViewModelFactory.java)
  at androidx.lifecycle.ViewModelProvider.get (ViewModelProvider.java)
  at androidx.lifecycle.ViewModelProvider.get (ViewModelProvider.java)
  at androidx.lifecycle.ViewModelLazy.getValue (ViewModelLazy.java)
  at androidx.lifecycle.ViewModelLazy.getValue (ViewModelLazy.java)
  at org.tiqr.core.authentication.AuthenticationConfirmFragment.getViewModel (AuthenticationConfirmFragment.java)
  at org.tiqr.core.authentication.AuthenticationConfirmFragment.onCreate (AuthenticationConfirmFragment.java)
  at androidx.fragment.app.Fragment.performCreate (Fragment.java)
  at androidx.fragment.app.FragmentStateManager.create (FragmentStateManager.java)
  at androidx.fragment.app.FragmentStateManager.moveToExpectedState (FragmentStateManager.java)
  at androidx.fragment.app.FragmentStore.moveToExpectedState (FragmentStore.java)
  at androidx.fragment.app.FragmentManager.moveToState (FragmentManager.java)
  at androidx.fragment.app.FragmentManager.dispatchStateChange (FragmentManager.java)
  at androidx.fragment.app.FragmentManager.dispatchCreate (FragmentManager.java)
  at androidx.fragment.app.Fragment.restoreChildFragmentState (Fragment.java)
  at androidx.fragment.app.Fragment.onCreate (Fragment.java)
  at androidx.navigation.fragment.NavHostFragment.onCreate (NavHostFragment.java)
  at androidx.fragment.app.Fragment.performCreate (Fragment.java)
  at androidx.fragment.app.FragmentStateManager.create (FragmentStateManager.java)
  at androidx.fragment.app.FragmentStateManager.moveToExpectedState (FragmentStateManager.java)
  at androidx.fragment.app.FragmentStore.moveToExpectedState (FragmentStore.java)
  at androidx.fragment.app.FragmentManager.moveToState (FragmentManager.java)
  at androidx.fragment.app.FragmentManager.dispatchStateChange (FragmentManager.java)
  at androidx.fragment.app.FragmentManager.dispatchCreate (FragmentManager.java)
  at androidx.fragment.app.FragmentController.dispatchCreate (FragmentController.java)
  at androidx.fragment.app.FragmentActivity.onCreate (FragmentActivity.java)
  at org.tiqr.core.base.BaseActivity.onCreate (BaseActivity.java)
  at org.tiqr.core.MainActivity.onCreate (MainActivity.java)
  at android.app.Activity.performCreate (Activity.java:8578)
  at android.app.Activity.performCreate (Activity.java:8557)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1384)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:4147)
```
### Motivation and Context
Fixes based on testing & Google Play console crash reporting.

### Testing Steps
For TIQR-327 check the login confirmation screen.
For TIQR-328, I could not reproduce this without faking long initializations. Only feasible verification is monitor crash logs in Play Console post release.

### Additional Information
⚠️ 🛑  **When the app integrate this core the custom application code must include the line `TiqrConfig.initialize(this)` in its initialization** ⚠️ 🛑
